README.md

```diff
@@ -1,26 +1,270 @@
+[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
+
 # 🛠️ release-toolkit
 
-[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
+The release toolkit is a series of small, composable tools that interact together to allow building heavily flexible release pipelines.
 
-Standard policy and procedure across the New Relic GitHub organization.
+The toolkit provides a series of commands, also available as GitHub Actions, that can compose with your existing pipelines to fully automate releases with meaningful, user-facing changelogs.
+
+The way in which the toolkit works is the following:
 
+1. Maintainers add human-readable changelog entries to the `## Unreleased` section in a [`CHANGELOG.md`](https://keepachangelog.com/en/1.0.0/) file.
+2. When the release process is initiated, a pipeline using the toolkit will:
+   1. Put the contents of the `## Unreleased` section in a machine-readable yaml file, `changelog.yaml`
+   2. Add dependency update entries from Renovate or Dependabot commits
+   3. Automatically calculate the next version following semver standards
+   4. Generate a Markdown document with the release notes for this release
+   5. Reincorporate all the contents of `changelog.yaml` into `CHANGELOG.md`, under the correct version header.
 
-## Install
+Each of these steps is performed as a different step in the pipeline, giving maintainers complete flexibility to skip or add intermediate steps. The machine-readable `changelog.yaml` can be easily edited with simple scripts, allowing more flexibility to add, remove, or change changelog entries before the steps that act on them take place.
 
-RT can be installed with Go by executing the following command:
+## Sneak peek
 
-```shell
-go install github.com/newrelic/release-toolkit@latest
+A pipeline using the release toolkit could look like the following:
+
+```yaml
```

Comment:
I've added an example yaml here, but at this point I think you have more expertise than I do on this :)

LMK if it makes sense to you @alvarocabanas @paologallinaharbur @gsanchezgavier

Reply:
I've checked and added bellow where it needs to be fixed.